common/app/conf/switches.scala

```diff
@@ -114,7 +114,13 @@ object Switches {
   val EnableOauthOnPreview = Switch("Performance", "enable-oauth-on-preview",
     "If this switch is switched on then the preview server requires login",
     safeState = On,
-    sellByDate = new LocalDate(2015, 3, 31)
+    sellByDate = new LocalDate(2015, 4, 30)
```

Comment:
Extending this switch while I am in here. There are some steps along the way to implementing PANDA and this is one of them.